tests/jax/models/example_model/model.py

```diff
@@ -16,16 +17,17 @@ def __init__(self) -> None:
             (1, 128),
         )
 
-        self.w0 = jax.numpy.ones(w0_shape)
-        self.w1 = jax.numpy.ones(w1_shape)
-        self.b0 = jax.numpy.ones(b0_shape)
-        self.b1 = jax.numpy.zeros(b1_shape)
+        self.w0 = random_tensor(w0_shape, minval=-0.01, maxval=0.01)
+        self.w1 = random_tensor(w1_shape, minval=-0.01, maxval=0.01)
+        self.b0 = random_tensor(b0_shape, minval=-0.01, maxval=0.01)
+        self.b1 = random_tensor(b1_shape, minval=-0.01, maxval=0.01)
 
     def __call__(
         self, act: jax.Array, w0: jax.Array, b0: jax.Array, w1: jax.Array, b1: jax.Array
     ) -> jax.Array:
         # Note how activations, weights and biases are directly passed to the forward
-        # method. `self` is not accessed.
+        # method as inputs, `self`` is not accessed. Otherwise they would be embedded
```

Comment:
Nitpick typo, two ` after self (wouldn't leave this comment if there weren't other unresolved comments)